src/util/strutil.h

```diff
@@ -0,0 +1,14 @@
+//
+// Created by alon on 07/10/2020.
+//
+
+#ifndef REDISGRAPH_STRUTIL_H
+#define REDISGRAPH_STRUTIL_H
+
+void str_tolower(const char *str, char *lower, short *lower_len);
+
+void str_toupper(const char *str, char *lower, short *lower_len);
```

Comment:
Add a short comment explaining the function and its arguments.
Also please rename the arguments accordingly.